java/runtime-common/src/main/java/org/ray/core/UniqueIdHelper.java

```diff
@@ -3,274 +3,40 @@
-  private static final ThreadLocal<Long> randSeed = new ThreadLocal<>();
-  private static final int batchPos = 0;
-  private static final int uniquenessPos = Long.SIZE / Byte.SIZE;
-  private static final int typePos = 2 * Long.SIZE / Byte.SIZE;
-  private static final BitField typeField = new BitField(0x7);
-  private static final int testPos = 2 * Long.SIZE / Byte.SIZE;
-  private static final BitField testField = new BitField(0x1 << 3);
-  private static final int unionPos = 2 * Long.SIZE / Byte.SIZE;
-  private static final BitField multipleReturnField = new BitField(0x1 << 8);
-  private static final BitField isReturnIdField = new BitField(0x1 << 9);
-  private static final BitField withinTaskIndexField = new BitField(0xFFFFFC00);
-
-  public static void setThreadRandomSeed(long seed) {
-    if (randSeed.get() != null) {
-      RayLog.core.error("Thread random seed is already set to " + randSeed.get()
-          + " and now to be overwritten to " + seed);
-      throw new RuntimeException("Thread random seed is already set to " + randSeed.get()
-          + " and now to be overwritten to " + seed);
-    }
-
-    RayLog.core.debug("Thread random seed is set to " + seed);
-    randSeed.set(seed);
-    rand.get().setSeed(seed);
-  }
-
-  public static Long getNextCreateThreadRandomSeed() {
-    UniqueID currentTaskId = WorkerContext.currentTask().taskId;
-    byte[] bytes;
-
-    ByteBuffer lbuffer = longBuffer.get();
-    // similar to task id generation (see nextTaskId below)
-    if (!currentTaskId.isNil()) {
-      ByteBuffer rbb = ByteBuffer.wrap(currentTaskId.getBytes());
-      rbb.order(ByteOrder.LITTLE_ENDIAN);
-      long cid = rbb.getLong(uniquenessPos);
-      byte[] cbuffer = lbuffer.putLong(cid).array();
-      bytes = MD5Digestor.digest(cbuffer, WorkerContext.nextCallIndex());
-    } else {
-      long cid = rand.get().nextLong();
-      byte[] cbuffer = lbuffer.putLong(cid).array();
-      bytes = MD5Digestor.digest(cbuffer, rand.get().nextLong());
-    }
-    lbuffer.clear();
-
-    lbuffer.put(bytes, 0, Long.SIZE / Byte.SIZE);
-    long r = lbuffer.getLong();
-    lbuffer.clear();
-    return r;
-  }
-
-  private static Type getType(ByteBuffer bb) {
-    byte v = bb.get(typePos);
-    return Type.values()[typeField.getValue(v)];
-  }
-
-  private static boolean getIsTest(ByteBuffer bb) {
-    byte v = bb.get(testPos);
-    return testField.getValue(v) == 1;
-  }
-
-  private static int getIsReturn(ByteBuffer bb) {
-    int v = bb.getInt(unionPos);
-    return isReturnIdField.getValue(v);
-  }
-
-  private static int getWithinTaskIndex(ByteBuffer bb) {
-    int v = bb.getInt(unionPos);
-    return withinTaskIndexField.getValue(v);
-  }
-
-  public static void setTest(UniqueID id, boolean isTest) {
-    ByteBuffer bb = ByteBuffer.wrap(id.getBytes());
-    setIsTest(bb, isTest);
+  public static UniqueID computeReturnId(UniqueID taskId, int returnIndex) {
```

Comment:
Let's add document for public methods in this class. 
I think it's okay to just copy from `src/ray/id.h`.